pkg/apis/contrail/v1alpha1/templates/webui_config.go

```diff
@@ -16,7 +16,12 @@ config.endpoints = {};
 config.endpoints.apiServiceType = "ApiServer";
 config.endpoints.opServiceType = "OpServer";
 config.regions = {};
-config.regions.RegionOne = "https://{{ .KeystoneIP }}:{{ .KeystonePort }}/v3";
+{{- /* HA core contrail tests are run without keystone, which causes invalid config generation. */}}
```

Comment:
I mean not only tests are run without keystone but also some production deployments. I suggest a change in this description

Reply:
Changed comment.